pkg/cmd/experimental/syncgroups/cli/syncgroups.go

```diff
@@ -143,10 +143,16 @@ func NewCmdSyncGroups(name, fullName string, f *clientcmd.Factory, out io.Writer
 	}
 
 	cmd.Flags().StringVar(&whitelistFile, "whitelist", whitelistFile, "path to the group whitelist file")
+	cmd.MarkFlagFilename("whitelist", "txt")
 	cmd.Flags().StringVar(&blacklistFile, "blacklist", whitelistFile, "path to the group blacklist file")
+	cmd.MarkFlagFilename("blacklist", "txt")
 	// TODO enable this we're able to support string slice elements that have commas
 	// cmd.Flags().StringSliceVar(&options.Blacklist, "blacklist-group", options.Blacklist, "group to blacklist")
+
 	cmd.Flags().StringVar(&configFile, "sync-config", configFile, "path to the sync config")
+	cmd.MarkFlagFilename("sync-config", "json", "yaml", "yml")
```

Comment:
That was scrapped. I'll amend.

On Thu, Nov 5, 2015 at 1:58 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/experimental/syncgroups/cli/syncgroups.go
> <https://github.com/openshift/origin/pull/5174#discussion_r44052729>:
>
> >  	cmd.Flags().StringVar(&configFile, "sync-config", configFile, "path to the sync config")
> > +	cmd.MarkFlagFilename("sync-config", "json", "yaml", "yml")
>
> I didn't realize we supported json sync configs today
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/5174/files#r44052729>.
>